### DISCUSSION TITLE
Current location

### DISCUSSION THREAD

**Discussion by OSMJaseB** [Zverik/every_door#524] [Q&A]
Is there a way to zoom/move the map to my current location?
That is often displayed in other apps by a target sort of icon. I think I have tried everything (it is a very clean interface, which is great!) and have not found it.
Thanks!
Also, I am on iOS and version 3.0 of Every Door.

**Comment by mnalis** [comment]
I dunno about iOS, sorry. On Android EveryDoor 3.0 the feature is there, however. To make it appear I have to:

- make sure GPS is turned on before starting the app
- make sure I'm in `Amenity` or `Micromapping` mode (e.g. `Notes` mode does not seem to have that icon)
- **not** already be on current location (i.e. pan a map around a little to make sure your map position is different then your current position).

The the `target` icon appears and can be clicked to position the map to current position:

![small_Screenshot_20230114_235206_info zverev ilya every_door](https://user-images.githubusercontent.com/156656/212500652-e84f39ef-6f52-4bae-84be-cc5d7b52bb86.jpg)

However if I disable GPS in Android, the `target` icon disappers. Turning GPS back on does **not** make it appear again, however, until I change EveryDoor mode (.e.g from `Amenity` to `Micomapping` or vice versa)

**Reply by OSMJaseB** [reply]
That makes sense. I am still not seeing the Target icon. Here's what I see, with GPS turned on.

![2023-01-15-09 57 59 720](https://user-images.githubusercontent.com/119752358/212549056-45b1cce7-4f85-4d2a-aa03-09b7fdc091b3.png)
